chef/cookbooks/pacemaker/recipes/default.rb

```diff
@@ -53,6 +55,24 @@
   include_recipe "corosync::default"
 end
 
+if (platform_family?("suse") && node.platform_version.to_f >= 12.0) || platform_family?("rhel")
```

Comment:
This is getting kind of ugly. Maybe it would be better to break this into smaller distro-specific recipes, then we can just ignore the rhel one.